python/lsst/ts/electrometer/commands.py

```diff
@@ -0,0 +1,576 @@
+from .enums import UnitMode, Filter, AverFilterType, readingOption
+
+
+class ElectrometerCommand:
+    """Class that contains commands to control the electrometer via RS-232.

```

Comment:
I think it might be clearer to say that it *formats commands for* the electrometer.

I was seriously confused at first, so I do hope you can clarify it.

Please also consider renaming it to something like `CommandFactory`.